ProcessLib/SmallDeformation/SmallDeformationFEM.h

```diff
@@ -209,6 +209,9 @@ class SmallDeformationLocalAssembler
         auto local_b = MathLib::createZeroedVector<NodalDisplacementVectorType>(
             local_b_data, local_matrix_size);
 
+        if (_process_data.isElementDeactivated(_element.getID()))
```

Comment:
I'd move the lookup into the constructor, since the element id is constant as well as the material ids and the `deactivated_material_ids`.

Reply:
Move it to the constructor.